### DISCUSSION TITLE
static ip address

### DISCUSSION THREAD

**Discussion by ebrennan114** [k8snetworkplumbingwg/multus-cni#1025] [General]
Is there a way to attach a static ip address to a pod using Multus ?

**Comment by maiqueb** [comment]
Yes!

Check the following example: https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/examples/macvlan-pod.yml

Relevant parts are:
- requesting IP capability: https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/examples/macvlan-pod.yml#L16
- requesting static IPAM: https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/examples/macvlan-pod.yml#L20
- requesting an IP from the Pod: https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/examples/macvlan-pod.yml#L46
